src/components/paper-time-input.js

```diff
@@ -49,6 +49,7 @@ export class PaperTimeInput extends PolymerElement {
             display: none;
           }
           --paper-input-container-shared-input-style_-_-webkit-appearance: textfield;
+          --paper-input-container-invalid-color: red;
```

Comment:
Maybe this styling should not be here?
Also, how to block saving the automation if the paper-time-input is invalid?

Reply:
You could move it to `ha-style.ts`

Reply:
OK, but how to add it such that it only applies to paper-input inside paper-time-input?